charts/spark-operator-chart/values.yaml

```diff
@@ -156,10 +156,14 @@ controller:
     #   memory: 300Mi
 
   # -- Security context for controller containers.
-  securityContext: {}
-    # runAsUser: 1000
-    # runAsGroup: 2000
-    # fsGroup: 3000
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 185
+    runAsGroup: 185
```

Comment:
Hardcoding UserID and GroupID feels a bit to static
If we ever decide to change our base image those could change

But requiring `runAsNonRoot`, setting an FsGroup and dropping all capabilities is a great idea

Reply:
Have removed the `runAsUser` and `runAsGroup` and pushed them down to the Dockerfile.